Example/PaymentSheet Example/PaymentSheet Example/PlaygroundController.swift

```diff
@@ -368,6 +370,29 @@ class PlaygroundController: ObservableObject {
         }
     }
 
+    var customPaymentMethodConfiguration: PaymentSheet.CustomPaymentMethodConfiguration? {
+        switch settings.customPaymentMethods {
+        case .on:
+            // Obtained from https://dashboard.stripe.com/settings/custom_payment_methods
+            let customPaymentMethodType = PaymentSheet.CustomPaymentMethodConfiguration.CustomPaymentMethodType(id: "cpmt_1QpId5Lu5o3P18ZpLwSqMXws",
```

Comment:
Maybe we have this vended from our merchant backend, and we add a parameter so we can test 1 or many custom payment methods? (e.g. the setting for customPaymentMethods is a value between 0-3 rather than a boolean)

Reply:
Hm, it feels needlessly complex tbh. Even if we needed multiple CPMs I'd rather have them defined client-side, this is more consistent with how merchants will integrate as well. Although nothing stopping them from using a backend to vend them as you suggested.